### PR TITLE
Message board final (edits/fixes from suggestions)

### PR DESCRIPTION
**Pull request by smile0711**

### CODE REVIEW

**Comment by smile0711** [comment]
@dburleson Don't forget to test under an user with admin access as well.

File: spa/src/App.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <main id="app" class="h-screen flex flex-col">
+  <main id="app" class="h-screen flex flex-col" v-if="this.$route.meta.wrapper">
```
Comment (dburleson) [inline spa/src/App.vue:2]: Remove this v-if

File: spa/src/App.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
     <div
       class="flex flex-none flex-row bg-lines justify-between h-full"
       style="height: 70px;"
```
Comment (dburleson) [inline spa/src/App.vue:5]: Add the v-if into the v-if on this element as `v-if="$store.data.isUser && this.$route.meta.wrapper"` Also on line 27

File: spa/src/App.vue [hunk 3]
```diff
@@ -105,6 +105,11 @@
     </div>
     <ModalRoot />
   </main>
+  <main id="app" v-else-if="!this.$route.meta.wrapper">
```
Comment (dburleson) [inline spa/src/App.vue:108]: Remove this entire new block `<main>` 108-112

**Review by dburleson — CHANGES_REQUESTED** [review]
Added notes on changes to be made to fix the chat/world loading bug. (see App.vue). After that, it's got the green light from me.

File: spa/src/App.vue (hunk 2 above)
Comment (dburleson) [reply spa/src/App.vue:5]: This is showing the wrong reference lines.

Lines to modify as discussed are line 6 and line 27 where there is currently `v-if="$store.data.isUser"`

**Comment by smile0711** [comment]
Changes made.  Good to ship.

**Pull request merged**